### ISSUE TITLE
DNSFilter blocks KDE Connect

### ISSUE THREAD

**Issue by Self-Perfection** [IngoZenz/personaldnsfilter#101]
KDE Connect is an app that allows controlling of PC from smartphone and vice versa.

It scans local network and connects to [TCP and UDP ports in range 1714:1764](https://userbase.kde.org/KDEConnect#I_have_two_devices_running_KDE_Connect_on_the_same_network.2C_but_they_can.27t_see_each_other).

Today I've noticed that DNSFilter prevented KDE Connect from working. I had to add it to whitelist to get it working again.

I currently use DNSFilter 1.50.41.0 from F-Droid.

**Comment by IngoZenz** [comment]
Did you check the pdnsf live log when you have the issue? Maybe KDE connect requires to connect some url which is blocked by a filter used in pdnsf?
Is it working when you deactivate blocking in pdnsf completely?

**Comment by Self-Perfection** [comment]
Straaange... I have not payed attention to the pdnsf log while I was having the issue. But I've checked with netcat running on my smartphone that I can't connect any port on 192.168.1.2. Then I've added KDEConnect to whitelist and the problem was gone so concluded it is pdnsf blocked the connection.

Yet right now pdnsf does not inhibit KDE Connect from working even if it is not in whitelist and connections from smartphone to 192.168.1.2 generally work.

I'm confused. I'm pretty sure my device was connected to the right network. Maybe reconnection or pdnsf rules update somehow resolved this?

I'm closing this for now. I'll try to collect more data if the issue appear again.
